experimental/FTheoryTools/src/AbstractFTheoryModels/methods.jl

```diff
@@ -793,6 +793,13 @@ Partially resolved global Tate model over a concrete base -- SU(5)xU(1) restrict
 ```
 """
 function resolve(m::AbstractFTheoryModel, resolution_index::Int)
+
+  # For model 1511.03209 and resolution_index = 1, a particular resolution is available from an artifact
+  if resolution_index == 1 && arxiv_id(m) == "1511.03209"
+    model_data_path = artifact"FTM-1511-03209"
+    return load(joinpath(model_data_path, "1511-03209-resolved.mrdi"))
```

Comment:
```suggestion
    model_data_path = artifact"FTM-1511-03209/1511-03209-resolved.mrdi"
    return load(model_data_path)
```
You can pass the filename to the artifact macro as well, and it handles the path joining and things for you (just my personal preference, no blocker whatsoever)